features/event-publisher/org.wso2.carbon.event.output.adapter.server.feature/src/main/resources/conf_templates/templates/repository/conf/output-event-adapters.xml.j2

```diff
@@ -161,4 +161,36 @@
     {% endif %}
     {% endfor %}
 
+    {% if output_adapter.azure is defined %}
```

Comment:
This has to be generic. We cannot say this is only for azure. We need a way to add any adapter and multiple adapters at a time

Reply:
Fixed with https://github.com/wso2/carbon-analytics-common/pull/778/commits/b0c5f02e4823bd0ce68b238e49c03f86eafb2396